UefiCpuPkg/CpuMpPei/CpuMpPei.c

```diff
@@ -504,8 +504,18 @@ InitializeExceptionStackSwitchHandlers (
 {
   EXCEPTION_STACK_SWITCH_CONTEXT  *SwitchStackData;
   UINTN                           Index;
+  EFI_STATUS                      Status;
+
+  Status = MpInitLibWhoAmI (&Index);
+
+  // MU_CHANGE [BEGIN] - CodeQL change
+  if (EFI_ERROR (Status)) {
+    DEBUG ((DEBUG_ERROR, "Failed to get processor number.  Aborting Exception Stack Switch handler setup.\n"));
+    return;

```

Comment:
Same thought about being in an exception handler

Reply:
Updated.